sentry/sentry.conf.example.py

```diff
@@ -66,7 +66,7 @@ def get_internal_network():
 
 # Instruct Sentry that this install intends to be run by a single organization
 # and thus various UI optimizations should be enabled.
-SENTRY_SINGLE_ORGANIZATION = True
+SENTRY_SINGLE_ORGANIZATION = env("SENTRY_SINGLE_ORGANIZATION", "false").lower() in ("true")
```

Comment:
```suggestion
SENTRY_SINGLE_ORGANIZATION = env("SENTRY_SINGLE_ORGANIZATION", "true").lower() in ("true")
```
oops

Reply:
If they enable multi-organization, they need to enable another feature flag though. See https://github.com/getsentry/sentry/blob/89479bb784838445845564e0fba06b19190aa573/src/sentry/features/temporary.py#L41-L42

Reply:
I'd also make it accept `1`, `yes`, `y`, and `t` too just to be comprehensive.